src/res/tree.html

```diff
@@ -1,60 +1,219 @@
+      }
+      .tree li.open > div:after {
+        display: none;
+      }
+      .tree li > div {
+        padding-left: 17px;
+        position: relative;
+        display: inline;
+        cursor: pointer;
+        color: black;
+        text-decoration: none;
+      }
+      .tree li > div:first-word {
+        font-weight: bold;
+      }
+      /* no children ul */
+      .o:after,
+      .o:before {
+        display: none !important;
+      }
+      .sc {
+        text-decoration-color: black;
+        font-weight: bold;
+        background-color: #d9d9d9;
+      }
+      input {
+        height: 36px;
+        padding: 0 14px;
+        border-radius: 4px;
+        background: white;
+        font-size: 16px;
+        border: 1px solid #2023281a;
+      }
+      button {
+        height: 36px;
+        border: none;
+        background: none;
+        border-radius: 4px;
+        cursor: pointer;
+        white-space: nowrap;
+      }
+      button:hover {
+        color: #00ac4f;
+      }
+      .button_outlined {
+        color: black;
+        font-size: 15px;
+        padding: 0 12px;
+        background: white;
+        border: 1px solid #2023281a;
+        display: flex;
+        align-items: center;
+        gap: 8px;
+      }
+      .total-count {
+        color: #5e5e5e;
+        font-weight: bold;
+        font-size: 14px;
+      }
+      .p {
+        font-weight: bold;
+      }
+      .header {
+        display: flex;
+        position: fixed;
+        top: 0;
+        left: 0;
+        gap: 16px;
+        flex-direction: row;
+        z-index: 100;
+        padding: 8px 20px;
+        background: #f5f5f5;
+        align-items: center;
+        box-sizing: border-box;
+        width: 100%;
+      }
+      .icon_gray {
+        color: #b7b7b7;
+      }
+      .header_actions {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        gap: 8px;
+        flex: 1;
+      }
+      .header-item {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        gap: 8px;
+      }
+      .view-type {
+        display: flex;
+        gap: 8px;
+        flex-direction: row;
+        font-weight: bold;
+        margin-top: 4px;
+        align-items: center;
+      }
+      .view-type_decoration {
+        display: flex;
+        gap: 8px;
+      }
+      .header-item_search {
+        gap: 8px;
+        flex: 1;
+      }
+      .search-input {
+        flex: 1;
+      }
+      .title {
+        font-size: 16px;
+      }
+      .wrapper {
+        padding: 70px 20px 10px;
+        background: white;
+        border-top: 1px solid #e9e9e9;
+      }
+
+      .child-count {
+        border: 1px solid #abb1c1;
+        border-radius: 3px;
+        padding: 0 5px;
+        color: #7d8089;
+      }
+
+      .t0,
+      .t1,
+      .t2,
+      .t3,
+      .t4,
+      .t5,
+      .t6 {
+        font-family: Söhne Mono, Monaco, Andale Mono, Ubuntu Mono, monospace !important;
```

Comment:
This is (kind of) a standard list of fonts used to display blocks of code.
Are you sure you cannot see "monospace" on Windows?

Reply:
I'll be happy to learn more if you provide a reference to some credible source.

As to `monospace`, it's a generic font family that translates to Consolas on Windows.

Reply:
Is a living example of reputable resources a sufficient resource for you? Here we have all different sets of mono fonts

Styles for a code block for [MSDN documentation ](https://learn.microsoft.com/en-gb/dotnet/csharp/tour-of-csharp/) 
<img width="1526" alt="image" src="https://github.com/async-profiler/async-profiler/assets/3423341/3f33d7ac-edc5-4bc6-bd8c-726cc3053575">

Styles for a code block in the Notion document
<img width="1479" alt="image" src="https://github.com/async-profiler/async-profiler/assets/3423341/724f299a-6765-4c04-b96f-24b32c897500">

also, if we ask chatGPT "what is the best set for the code block in web" we will have that answer
```
code {
  font-family: Monaco, Consolas, "Courier New", monospace;
}
```

I use a little bit different set, but will be happy to change it for MSDN, notion or the last option if you prefer!

Reply:
Thanks for the references. The font list from MSDN and Notion makes much more sense to me: 1) these fonts are widely available; 2) the list includes Windows fonts; 3) there are no strange names with non-ASCII characters :)

Reply:
BTW, since you've mentioned ChatGPT - I would not be able to accept a PR if it has any content generated by ChatGPT or similar.

To contribute to async-profiler project, the contributor must be the author of the work and must have all rights and permissions to submit the work under Apache 2.0 License.

Reply:
Make sence! No, nothing in that PR generated by ChatGPT or similar, you can see even my list of fonts is different than it's list.